### PR TITLE
US11636 - Angular CLI Documentation

### PR DESCRIPTION
**Pull request by seancdavis**
Update README with angular cli usage and fix outdated instructions

---

Corresponds to crdschurch/crds-styleguide#282

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,135 @@
...
+optionally override any of the customizable Bootstrap variables prior to
+import.
 
-Once you've added `crds-styles` to `package.json` you need to import the stylesheet in your application's main '.scss' file using the following convention. You can optionally override any of the customizable Bootstrap variables prior to import...
+```scss
+@import '~crds-styles/assets/stylesheets/bootstrap';
+```
+
+_Note: The tilde character (`~`) assumes use of the
+[node-sass](https://github.com/sass/node-sass) library.
```
Comment (tcmacdonald) [inline README.md]: Is this accurate? 

It's my understanding that the tilde character convention for resolving to `node_modules/` is provided via [Webpack's sass-loader](https://github.com/webpack-contrib/sass-loader#imports) plugin. I don't believe `node-sass` supports this feature out-of-the-box but you can use [node-sass-tilde-importer](https://github.com/matthewdavidson/node-sass-tilde-importer) package to emulate the same behavior.

File: README.md [hunk 2]
```diff
@@ -1,88 +1,135 @@
...
+sure your `.angular-cli.json` file copies the assets directory:
+
+```js
+{
+  "apps": [{
+    "assets": [
+      "assets"
+    ]
+  }
+]
```
Comment (tcmacdonald) [inline README.md]: This approach is not what I would've expected. Is there no way to copy files directly from `./node_modules/crds-styles/assets/` into our `dist/assets/` directory?

File: README.md (hunk 1 above)
Comment (seancdavis) [reply README.md]: You're right. Good catch. Thanks.

File: README.md (hunk 2 above)
Comment (seancdavis) [reply README.md]: Yeah, I thought this was simpler, but I agree—there's less room for error with your example.

File: README.md [hunk 3]
```diff
@@ -1,88 +1,142 @@
-# crds-styles
+crds-styles
+==========
 
-`crds-styles` provides the primary CSS framework and global style patterns that power [crossroads.net](http://crossroads.net) and it's related properties. Any elements and/or implementation details included within the [Digital Design Kit (DDK)](http://github.com/crdschurch/crds-styleguide) are provided by this NPM package. Any deviations from the patterns defined there should be implemented in the client application.
+`crds-styles` provides the primary CSS framework and global style patterns that
+power [crossroads.net](http://crossroads.net) and it's related properties.
```
Comment (zarahzachz) [inline README.md]: 🚨 Grammar error: Should be 'its' because it's possessive, not a contraction of 'it is'

File: README.md [hunk 4]
```diff
@@ -1,88 +1,142 @@
...
-### Option #1 - Symlink Local Directory
+### Option #1 (Recommended) - External Build
 
-Assuming the `crds-styles` repo resides in the same local directory as your project, you can do the following to setup the symlink...
+The easiest way to include crds-styles is by loading the external CSS file. The
+build is deployed to S3 and is available through our Cloudfront distribution.
 
-    $ cd path/to/your/project-directory
-    $ npm link ../crds-styles
+This approached is most useful when you don't need to override any styles, or
```
Comment (zarahzachz) [inline README.md]: 'approached' shouldn't be past-tense.

File: README.md (hunk 3 above)
Comment (seancdavis) [reply README.md]: Thank you for explaining that to me.

**Pull request merged**